### DISCUSSION TITLE
I built a new tailwindcss compiler with Typescript.

### DISCUSSION THREAD

**Discussion by voorjaar** [tailwindlabs/tailwindcss#3280] [Show and tell]
I built a new tailwindcss compiler with Typescript called [windicss](https://github.com/voorjaar/windicss). It runs fast, can build css in real-time during development, well-matched with vue and svelte. It has native support of cross-browser and minimized build, and has supported all the utilities and variants of tailwindcss. Give it a try. If you like leave a star, or send some feedback if you find problems. Pull requests are most helpful.

**Comment by skyrpex** [comment]
I'm using it in Vite through https://github.com/windicss/vite-plugin-windicss and what can I say... It's awesome! It really goes well with esbuild's philosophy.

I think tailwindcss should merge efforts with windicss, somehow.

**Comment by florianbouvot** [comment]
@voorjaar excellent, is there a way to use it in Webpack, Gulp or PostCSS?

**Reply by voorjaar** [reply]
If you use JavaScript frameworks, such as vue and svelte, you can use preprocessor directly. Regarding whether to support other bundlers, we are considering this now. Of course, if you are willing to develop a plugin for us, it will be very helpful.
